README.md

```diff
@@ -1,76 +1,61 @@
-## THREEFOLD GRID DASHBOARD
+# TFGrid_Dashboard
 
 ![GitHub release (latest by date)](https://img.shields.io/github/v/release/threefoldtech/tfgrid_dashboard)
+[![Build](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/build.yml/badge.svg)](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/build.yml)
+[![Lint](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/lint.yaml/badge.svg)](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/lint.yaml)
 [![Selenium _Tests](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/Selenium.yaml/badge.svg)](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/Selenium.yaml)
 [![Cypress Tests](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/Cypress.yaml/badge.svg)](https://github.com/threefoldtech/tfgrid_dashboard/actions/workflows/Cypress.yaml)
 
-A unified application where users can do the following and so much more !
+## Introduction
 
-- View and search their connected polkadot.js accounts
-- Connect/Disconnect from polkadot.js
-- View balances on the TF Chain
-- Buy TFT
-- Transfer TFT within the TFChain
-- Swap TFT to and from Cosmos and Stellar
-- Create/delete/edit/view/search twins, farms, farm public IPs, farm nodes and farm nodes public configuration.
-- Add Stellar payout to farms
-- Reserve/Unreserve dedicated nodes.
-- Explore the grid capacity (i.e view the TF Grid statistics, nodes and farms across the world).
+The Dashboard is a unified admin interface for everything related to Threefold Grid such as Portal, Explorer, Calculators, Zero-OS Bootstrap & Playground. For more details, check [Dashboard Manual](https://library.threefold.me/info/manual/#/manual__dashboard_readme)
 
-If you don't know where to start, you can also review the TF Chain manual before even connecting to polkadot.js.
+- **Clone the repository**
```

Comment:
I think this section is misplaced

Reply:
removed from README as it's already included in getting_started doc